Backend/app/rider-platform/rider-app/Main/spec/Storage/ParcelDetails.yaml

```
@@ -0,0 +1,28 @@
+  constraints:
+    searchRequestId: PrimaryKey
+
+  excludedDefaultQueries:
+    - createMany
+    - updateByPrimaryKey
+    - findByPrimaryKey
+
+  queries:
+    findBySearchRequestId:
+      kvFunction: findOneWithKV
+      where: searchRequestId
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)